src/components/TaxPayer/PersonFields.tsx

```diff
@@ -22,7 +22,8 @@ import ListItemText from '@material-ui/core/ListItemText'
 import PersonIcon from '@material-ui/icons/Person'
 
 export const labels = {
-  fname: 'First Name and Initial',
+  fname: 'First Name',
+  initial: 'Initial',
```

Comment:
Would this be more clear if it was called 'Middle Initial'?

Reply:
Agreed, `midInitial: 'Middle Initial'`  would likely be better naming for the key and the label.